pillar/params.sls

```diff
@@ -58,6 +58,10 @@ flannel:
   etcd_key:       '/flannel/network'
   iface:          'eth0'
 
+proxy:
+  http:           ''
+  https:          ''
```

Comment:
no_proxy will also be required, as there are many environments where HTTP proxies are needed for external things (e.g. docker hub), but won't work for internal things (e.g. a local registry).